src/main/scala/slamdata/SbtSlamData.scala

```diff
@@ -282,5 +357,20 @@ object SbtSlamData extends AutoPlugin {
         currentVersion + "-" + "git rev-parse HEAD".!!.substring(0, 7)
       else
         currentVersion
-    })
+    },
+
+    Test / test := (Def.taskDyn {
+      val t: Unit = (Test / test).value
+      if (name.value == "root" && isTravisBuild.value) {
+        Def.task {
+          versionCheck.value
+          t
```

Comment:
Btw, I think you can avoid the slight oddness of "also running the tests" by inlining things here. Basically, the `if`/`else` would be:

```scala
if (..) {
  Def task {
    versionCheck.value
    (Test / test).value
  }
} else {
  Test / test
}
```

Nbd either way.

Reply:
That's not how dynamic tasks work. That wouldn't type check.